pydicom/pixel_data_handlers/gdcm_handler.py

```diff
@@ -0,0 +1,110 @@
+import sys
+have_numpy = True
+try:
+    import numpy
+except ImportError:
+    have_numpy = False
+    raise
+
+have_gdcm = True
```

Comment:
I think caps for globals is useful, since it tells me not to mess with their values and lets me know to look beyond local scope when trying to follow the code. Within a module it also implies that the variable is public (contrast with the `can_use_gdcm` variable on line 15, although that's probably better off being `_can_use_gdcm` or being defined within `get_pixeldata`).